src/token.ts

```diff
@@ -153,3 +153,10 @@ export class mIOToken extends PositiveFiniteInteger {
     return new IOToken(this.valueOf() / MIO_PER_IO);
   }
 }
+
+export type AoSigningFuction = (args: {
```

Comment:
Perhaps this should be called AoSigner, considering it's returned from createAoSigner()